beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -567,11 +567,13 @@ impl<E: EthSpec> PeerManager<E> {
                     Protocol::BlocksByRange => PeerAction::MidToleranceError,
                     Protocol::BlocksByRoot => PeerAction::MidToleranceError,
                     Protocol::BlobsByRange => PeerAction::MidToleranceError,
+                    // TODO(lc-update) review this comment and logic
```

Comment:
Looks good to me, we may implement a light-client "client" in the future, and add the boilerplate then